packages/bsky/src/api/app/bsky/actor/getProfile.ts

```diff
@@ -52,36 +50,29 @@ const skeleton = async (
   params: Params,
   ctx: Context,
 ): Promise<SkeletonState> => {
-  const { actorService, modService } = ctx
-  const { canViewTakendownProfile } = params
+  const { actorService } = ctx
+  const { canViewTakedowns } = params
   const actor = await actorService.getActor(params.actor, true)
   if (!actor) {
     throw new InvalidRequestError('Profile not found')
   }
-  if (!canViewTakendownProfile && softDeleted(actor)) {
-    const isSuspended = await modService.isSubjectSuspended(actor.did)
-    if (isSuspended) {
-      throw new InvalidRequestError(
-        'Account has been temporarily suspended',
-        'AccountTakedown',
-      )
-    } else {
-      throw new InvalidRequestError(
-        'Account has been taken down',
-        'AccountTakedown',
-      )
-    }
+  if (!canViewTakedowns && softDeleted(actor)) {
+    // @TODO throw a different error if the accoutn is suspended
```

Comment:
we don't have the info in the appview for this with the current architecture anymore - would be easier to do with labels

Reply:
One idea is to smuggle it into the takedown reference: e.g. something like `5917` vs. `5917-SUSPEND`.

Reply:
yeah i thought of that... kinda hacky but i suppose it works 🤔

Reply:
If it's not all that important to be able to provide this error message then maybe we drop it.  But if it's important, might be worth the hack?

Reply:
alrighty did the hack - I ended up adding a bit more info to the refs as well so they're not just opaque numbers. now they look like `BSKY-TAKEDOWN-5917` or `BSKY_SUSPEND-5917`. makes it look less hacky 😛